src/components/settings/LocationSettings.js

```diff
@@ -93,41 +89,43 @@ export const LocationSettings = () => {
     locations = [getLocationMarker(defaultAlternativePosition)];
   }
 
+  const handleMapPress = ({ nativeEvent }) => {
+    setSelectedPosition({
+      ...nativeEvent.coordinate
+    });
+  };
+
+  const handleSave = () => {
+    if (selectedPosition) {
+      setAndSyncLocationSettings({
+        alternativePosition: geoLocationToLocationObject(selectedPosition)
+      });
+    }
+    setSelectedPosition(undefined);
```

Comment:
why is this set to undefined if there is a selected position a line before?

Reply:
This code is copied and pasted into the `handleSave` function. The previous version of the code is protected.

Reply:
@d4v3d4vE do you know?

Reply:
ChatGPT's explanation:
> The line setSelectedPosition(undefined); is setting selectedPosition to undefined to reset or clear the selected position after the user saves or aborts the operation. This is a common practice to ensure that the state is reset, preventing any residual state from interfering with subsequent operations.


personally I couldn't explain it on the fly

Reply:
so it is used first for alternative position and cleared afterwards, right? maybe the collapsible is closed with unsetting selected position? what happens if we do not set it to undefined?

Reply:
yes, you are correct. the selectedPosition is used to update the alternativePosition in the location settings and is then cleared to reset the state

the collapsible component's visibility is controlled by showMap, which is set to false immediately after clearing the selected position

but I also just checked manually various cases and I couldn't finde any error wich should appear when we are not setting `setSelectedPosition(undefined);`, therefore I'd say that we can get rid of it